src/08-tasks/tasks.apib

```diff
@@ -8,7 +8,7 @@ Get a list of tasks.
 
     + Attributes (object)
         + filter (object, optional)
-            + assignee_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional)
+            + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional)
```

Comment:
Let's make this filter similar to the one on invoicing https://developer.teamleader.eu/#/reference/deals/deals/deals.list

So a `assignee` filter with a type and id.

Reply:
This isn't filtering by "assignee" though, it also returns tasks that are assigned to a team to which you belong to. This is consistent with how `events.list` works, with a `user_id` filter that returns events for which you are attending.

Reply:
If we want to add a filter to search on assignee with type/id, how will that relate to this `user_id`, and will that not be confusing for the people using it?

Reply:
> If we want to add a filter to search on assignee with type/id, how will that relate to this `user_id`, and will that not be confusing for the people using it?

The idea was that that would be a different filter, but not sure that makes sense.

What we need is a filter that returns both tasks that are assigned to you AND tasks that are assigned to a team to which you belong to. You should be able to get all those tasks in one request.

I think both this and the "filter to search on assignee with type/id" are valid use cases, but for the moment we only need the first one. 

Not sure what to do.

Reply:
If you think it does not make sense to use the assignee filter we can use the `user_id` with a proper comment next to it. We can still make a breaking change later I guess.

Reply:
We'll add a comment to make clear what the `user_id` filter does.